src/2_banner.js

```diff
@@ -144,11 +144,15 @@ banner = function(branch, options, linkData, storage) {
 				sendSMS(doc, branch, options, linkData);
 			});
 		}
+
 		var closeButton = doc.getElementById('branch-banner-close');
+		banner.close = function() {
```

Comment:
I like the idea of the banner just returning the close method. If we want the banner to have other methods in the future, we can just have the banner return an object of functions, and store that object on branch. I'll just have it return the close function for now.